### ISSUE TITLE
Icons for Tricolor Sprite, Overdrive, and Dominant Harmony are missing

### ISSUE THREAD

**Issue by nagi-desuuu** [summertriangle-dev/sparklebox#67]
Upon sorting cards by skills, icons are missing for the skills mentioned in the title.
![Screenshot (274)](https://github.com/summertriangle-dev/sparklebox/assets/75596775/b07297f7-4d96-4c45-ab2f-0497b7ddc0fc)
![Screenshot (275)](https://github.com/summertriangle-dev/sparklebox/assets/75596775/7cf21359-ade4-4943-b816-fce225794fab)

**Comment by toadsworth-jr** [comment]
I would like to add on that `Dominant Harmony` is now also in this list as it also lacks an icon.

![image](https://github.com/summertriangle-dev/sparklebox/assets/75596775/3f129e90-882a-40ce-b16c-efde32a7fc8b)
![image](https://github.com/summertriangle-dev/sparklebox/assets/75596775/0c8f6dee-0984-45ea-89b8-b0e1091f6d95)

**Comment by summertriangle-dev** [comment]
Yeah, I lost the template I used for making those icons. I'll get around to making a new one someday.
